src/components/Receive.jsx

```diff
@@ -71,6 +73,7 @@ const Receive = ({ currentWallet }) => {
           onChange={(e) => setAccount(parseInt(e.target.value, 10))}
           style={{ maxWidth: '21ch' }}
           required
+          disabled={!settings.useAdvancedWalletMode}
```

Comment:
If in "magic wallet mode", don't let the user choose any other mix depth than 0️⃣  for receiving.

Reply:
Great idea! Have you thought about not displaying the select box in the first place (in this mode)?

Reply:
I thought about it yeah. We should probably revamp both the send / receive screens a bit according to Figma designs and improve wording as well as things such as hiding accounts when they're not needed. Let's talk about that in today's call!

Reply:
I tracked https://github.com/joinmarket-webui/joinmarket-webui/issues/57.